src/App.tsx

```diff
@@ -3,16 +3,15 @@ import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
 import BlogList from './components/List';
 import BlogPostPage from './components/Post';
 import BlogPostProps from './types/BlogPostProps';
+import { fetchAPI } from './constants';
```

Comment:
Keeping things DRY (Don't Repeat Yourself) I saw that `fetchAPI` was re-used and so I abstracted it into a constants file